src/helpers/profilePic.js

```diff
@@ -0,0 +1,26 @@
+import multer from 'multer';
+import cloudinary from 'cloudinary';
+import cloudinaryStorage from 'multer-storage-cloudinary';
+import dotenv from 'dotenv';
+import config from '../db/config/config';
+
+const { cloudName, apiKey, apiSecret } = config;
+
+
+dotenv.config();
+
+cloudinary.config({
+  cloud_name: cloudName,
+  api_key: apiKey,
+  api_secret: apiSecret,
+});
+
+const storage = cloudinaryStorage({
+  cloudinary,
+  folder: 'avatars',
+  allowedFormats: ['jpg', 'png', 'jpeg'],
+});
+
+const upload = multer({ storage });
```

Comment:
Can you please include an inline comment that explains what's going on in this file